submarine-cloud-v2/helm-charts/submarine-operator/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
Need to add the license

Reply:
@pingsutw I have added the license, thank you.